### ISSUE TITLE
Use no cookies version of youtube iframe

### ISSUE THREAD

**Issue by SamWindell** [sharu725/youtube-embed#33]
It would be great to use the `youtube-nocookie.com` of the embed because it would remove the need to get cookie-consent at all. 

This plugin already solves part of the problem - there is no initial need to get cookie consent until the embed is clicked on. However, in the current version that uses the regular embed, cookies are added as soon as the youtube video is clicked. As I understand it, it is the responsibility of the container website to ask permissions to use cookies. However, if the `youtube-nocookie.com` version is used, no cookies are added, even when the video is started. And therefore there is no need to ask for cookie consent.

To test this, I embedded the normal iframe and watched my browser's Developer Tools storage window. Cookies we added when the video started.

Normal cookie-enabled iframe (similarly used by this plugin):
```
<iframe 
width="560" 
height="315" 
src="https://www.youtube.com/embed/dQw4w9WgXcQ?si=8dtn2gTcF4Q9xFdH" 
title="YouTube video player" 
frameborder="0" 
allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" 
referrerpolicy="strict-origin-when-cross-origin" 
allowfullscreen></iframe>
```

In comparison if I used `youtube-nocookie.com` version, no cookies were added even when the link was clicked:
```
<iframe 
width="560" 
height="315" 
src="https://www.youtube-nocookie.com/embed/dQw4w9WgXcQ?si=8dtn2gTcF4Q9xFdH" 
title="YouTube video player" 
frameborder="0" 
allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" 
referrerpolicy="strict-origin-when-cross-origin" 
allowfullscreen></iframe>
```

As far as I can tell, the only thing that would need to change in this package is replacing `youtube.com/embed` with `youtube-nocookie.com/embed/`.

This was previously mentioned at #26 but there's more to this.

**Comment by sharu725** [comment]
Thank you. I might have forgotten. Let me get this done.

**Comment by sharu725** [comment]
Fixed in ``svelte-youtube-embed@0.3.2``

**Comment by SamWindell** [comment]
Fantastic, thank you
